specs/job_validation.adoc

```diff
@@ -0,0 +1,200 @@
+// vim: tw=79
+
+= Definition file and job validation
+
+As there where lot changes in the sds schema, The job-validation module which
+is in the common project needs few changes or enhancement in order to validate
+the sds definition file and the incomming jobs which contains input values.
+
+== Problem description
```

Comment:
I feel problem description and proposed changes are mixed up here. Move the proposed changes to respective section.